src/translator/response.h

```diff
@@ -26,21 +26,23 @@ struct Point {
 /// Alignment is a sparse matrix, where Points represent entries with values.
 typedef std::vector<Point> Alignment;
 
-/// -loglikelhoods of the sequence components as proxy to quality.
-struct Quality {
-  /// Certainty/uncertainty score for sequence.
-  float sequence;
-  /// Certainty/uncertainty for each word in the sequence.
-  std::vector<float> word;
-};
-
 /// Response holds AnnotatedText(s) of source-text and translated text,
 /// alignment information between source and target sub-words and sentences.
 ///
 /// AnnotatedText provides an API to access markings of (sub)-word and
 /// sentences boundaries, which are required to interpret Quality and
 /// Alignment (s) at the moment.
 struct Response {
+  /// WordsQualityEstimate contains the quality data of a given translated sentence.
+  /// It includes the confidence (proxied by a probability) of each decoded word
+  /// (higher probabilities imply better-translated words), the ByteRanges of each term,
+  /// and the probability of the whole sentence, represented as the mean word scores.
+  struct WordsQualityEstimate {
+    std::vector<float> wordQualityScores;
+    std::vector<ByteRange> wordByteRanges;
+    float sentenceScore = 0.0;
+  };
```

Comment:
Brevity and suitability will be appreciated here. Someone at JS will be writing a lot these once bindings are written, [you don't want them screaming at you](https://news.ycombinator.com/item?id=28325401). 

Here's a recommendation:

```cpp
// For an external user this means a sentence, not Words - you have a sentenceScore anyway. 
struct SentenceQualityEstimate {
	std::vector<ByteRange> wordByteRanges;
	// wordScores, Quality is known from struct or var naming.
	// The following is also consistent with sentenceScore.
	std::vector<float> wordScores; 
	float sentenceScore;
};
```

Reply:
We renamed it. Commit c0c1bbc